x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/alerts_page.tsx

```diff
@@ -137,9 +138,9 @@ function AlertsPage() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  // In a future milestone we'll have a page dedicated to rule management in
-  // observability. For now link to the settings page.
-  const manageRulesHref = prepend('/app/management/insightsAndAlerting/triggersActions/alerts');
+  const manageRulesHref = config.unsafe.rules
+    ? prepend('/app/observability/rules')
+    : prepend('/insightsAndAlerting/triggersActions/alerts');
```

Comment:
Just curious but why does this part not use the same hook? Feels like duplicating knowledge

Reply:
@mgiota, yeah, it looks the same as the hook https://github.com/elastic/kibana/pull/124132/files#diff-b87086a406bae7ecb828ec2851aa67fe5014e8627f9665c6b67adac63892790aR9